nexus/src/app/saga.rs

```diff
@@ -56,70 +87,89 @@ pub(crate) fn create_saga_dag<N: NexusSaga>(
     Ok(SagaDag::new(dag, params))
 }
 
-impl super::Nexus {
-    pub(crate) async fn sagas_list(
-        &self,
-        opctx: &OpContext,
-        pagparams: &DataPageParams<'_, Uuid>,
-    ) -> ListResult<nexus_types::internal_api::views::Saga> {
-        // The endpoint we're serving only supports `ScanById`, which only
-        // supports an ascending scan.
-        bail_unless!(
-            pagparams.direction == dropshot::PaginationOrder::Ascending
-        );
-        opctx.authorize(authz::Action::Read, &authz::FLEET).await?;
-        let marker = pagparams.marker.map(|s| SagaId::from(*s));
-        let saga_list = self
-            .sec_client
-            .saga_list(marker, pagparams.limit)
-            .await
-            .into_iter()
-            .map(nexus_types::internal_api::views::Saga::from)
-            .map(Ok);
-        Ok(futures::stream::iter(saga_list).boxed())
-    }
+/// External handle to a self-contained subsystem for kicking off sagas
```

Comment:
Nit:

```suggestion
/// Handle to a self-contained subsystem for kicking off sagas
```

(This is in the nexus crate and used by Nexus, it doesn't feel particularly "external")

Reply:
Done.